### PR TITLE
Development weblets docs

### PR DESCRIPTION
**Pull request by maayarosama**
### Description

Renaming files in playground docs
re-adding sections in weblets docs

### Related Issues
#217
#120

### CODE REVIEW

File: src/playground/wallet_connector.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Profile Manager
```
Comment (AhmedHanafy725) [inline src/playground/wallet_connector.md:1]: can we rename it inside the file to wallet connector too?

**Review by AhmedHanafy725 — CHANGES_REQUESTED** [review]

File: src/playground/k8s.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Kubernetes is the standard container orchestration tool.
 
 On the TF grid, Kubernetes clusters can be deployed out of the box. We have implemented [K3S](https://k3s.io/), a full-blown Kubernetes offering that uses only half of the memory footprint. It is packaged as a single binary and made more lightweight to run workloads in resource-constrained locations (fits e.g. IoT, edge, ARM workloads).
 
-- Make sure you have an activated [profile](./weblets_profile_manager.md) 
+- Make sure you have an activated [profile](./wallet_connector.md)
```
[the remark this answers is not in the document]
Comment (AhmedHanafy725) [reply src/playground/k8s.md:7]: this should be applied to all files

File: src/playground/caprover.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ It has following benefits :
 ### Requirements
 
 - You need an account on TF-Chain, and there needs to be TFT on the account (see getting started)
-- [Make sure your profile is activated](./weblets_profile_manager.md)
+- [Make sure your profile is activated](./wallet_connector.md)
```
Comment (AhmedHanafy725) [inline src/playground/caprover.md:31]:
```suggestion
- [Make sure your wallet is connected](./wallet_connector.md)
```

**Comment by Mik-TF** [comment]
I adjusted some inner TOC links, like sys admins TOC and others.

**Review by Mik-TF — APPROVED** [review]

**Comment by Mik-TF** [comment]
@AhmedHanafy725 
It seems that SSH wasn't in the summary.md.

I added it in summary.md, and the toc for playground and sys admin.

Let me know if you want to put it elsewhere.
ssh_menu

**Comment by Mik-TF** [comment]
Screen Shot 2023-08-02 at 6 22 45 PM

Maybe it would be better to have a SSH section in Getting Started, and have 

* SSH
   *  SSH Remote Connection
   * SSH on WSL

**Comment by Mik-TF** [comment]
I re-organized the SSH section to present the 3 main methods, openssh, putty and wsl.

**Review by Mik-TF — APPROVED** [review]
All the changes should be there.

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**